src/idl_gen_grpc.cpp

```diff
@@ -35,8 +37,7 @@ class FlatBufMethod : public grpc_generator::Method {
  public:
   enum Streaming { kNone, kClient, kServer, kBiDi };
 
-  FlatBufMethod(const RPCCall *method)
```

Comment:
please refrain from making formatting changes unrelate to your change, makes it much harder to review.